Incopo/Controller/RegisterViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  RegisterViewController.swift
+//  Incopo
+//
+//  Created by Laurentiu Ile on 09.02.2021.
+//
+
+import UIKit
+import Firebase
+
+class RegisterViewController: UIViewController {
+    
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var firstNameTextField: UITextField!
+    @IBOutlet weak var lastNameTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var confirmPasswordTextField: UITextField!
+    
+    let decimalCharacters = CharacterSet.decimalDigits
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.hideKeyboardWhenTappedOutside()
+    }
+    
+    @IBAction func singUpPressed(_ sender: Any) {
+        
+        let user = User(email: emailTextField.text,
+                        firstName: firstNameTextField.text,
+                        lastName: lastNameTextField.text,
+                        password: passwordTextField.text,
+                        recents: [],
+                        profilePicture: UIImage())
+        
+        let error = validateUser(user: user)
+        if let error = error {
+            ICPAlert.showAlert(on: self, with: "Registration failed".localized, message: error)
+            return
+        }
+        
+        performRegistration(user: user)
+    }
+    
+    func validateUser(user: User) -> String? {
+        var error: String?
+        
+        let firstNameContainsDecimals = user.firstName?.rangeOfCharacter(from: decimalCharacters)
+        if firstNameContainsDecimals != nil {
```

Comment:
it would be better to directly allow only letters in the text field and ignore everything else